css-four-column-layout/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Four Column Layout</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <main class="row flex-wrap justify-content-center">
+    <h2 class="full-width align-text">How It Works</h2>
+    <section class="column-quarter row flex-direction-column">
```

Comment:
so instead of using the `row` class to assign `display-flex` I would have a `display-flex` class and use that as needed. You only want one row class per row, and for this HTML it looks like that row should surround all of the sections with column classes.